### ISSUE TITLE
Starting from v4.46, the `logits` model output will have the same type as the model (except at train time, where it will always be FP32)

### ISSUE THREAD

**Issue by cuichenxu** [RUC-NLPIR/FlashRAG#89]
我在运行 [https://github.com/RUC-NLPIR/FlashRAG/blob/main/docs/introduction_for_beginners_zh.md](url) 的demo时，出现如下错误
![image](https://github.com/user-attachments/assets/230cd103-2c49-4f02-bb1b-530e562c5147)
有人遇到过类似的错误吗？

**Comment by ignorejjj** [comment]
这应该只是一个warning，能放出完整的报错信息吗

**Comment by cuichenxu** [comment]
> 这应该只是一个warning，能放出完整的报错信息吗

![image](https://github.com/user-attachments/assets/a10eb342-5647-4ad3-bd52-0ab85164d857)
这是完整的报错信息。

**Comment by ignorejjj** [comment]
应该是sample导致的溢出，可以在`generation_params`里面设置`do_sample=False`.

**Comment by cuichenxu** [comment]
> 应该是sample导致的溢出，可以在`generation_params`里面设置`do_sample=False`.

感谢感谢，解决了。

**Comment by clariceluvyyyya** [comment]
最后是怎么解决的啊？generation_params在哪

**Comment by ignorejjj** [comment]
config文件里面有
